### PR TITLE
Mada style update

### PR DESCRIPTION
**Pull request by jskov**
This patch:

* Updates the comparison for 0.9.9: please re-run it. It works for me.
* Adds buildspec for version 1.0.0

Cheers!

### CODE REVIEW

**Comment by hboutemy** [comment]
> Updates the comparison for 0.9.9: please re-run it. It works for me.

It seems you are lucky with 0.9.9: your build got a patch level of JDK that generates the same JavaScript in Javadoc. I was not as lucky as you. But IIUC, you were as unlucky as me with 0.9.8 :)

this is a global weakness for environment spec to rebuild javadoc that expands to `.module` files generated by Gradle: #193 created to ignore these `.module` files...

I'll merge your lucky build as part of this PR

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-0.9.9.buildinfo [hunk 1]
```diff
@@ -25,7 +25,7 @@ outputs.2.1.checksums.sha512=c40147bc923c2fb920f48a20904208f1a5a66b933bcafc4b1da
 
 outputs.2.2.filename=mada-style-gradle-0.9.9.module
 outputs.2.2.length=4103
-outputs.2.2.checksums.sha512=5537207a6e2f4dc1fa8bff009ded4730b87b19145c207867247a7451f3777749774c3d304ecadd3b0427125bc608ca1b7c9c7cb616bf21788ca7ff36273e6df8
```
Comment (hboutemy) [inline content/dk/mada/style/mada-style-gradle/mada-style-gradle-0.9.9.buildinfo:28]: you'll need to rm the `.diffoscope` file too, and comment it in the `.buildspec`

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+# Rebuild command
+command="bash ./gradlew --no-daemon javadoc publish publishToMavenLocal -Pversion=${version}"
+#command="bash ./gradlew --no-daemon clean assemble generateBuildInfo publishToMavenLocal -Pversion=${version}"
+
+# Location of the buildinfo file generated during rebuild (by attifact:compare for Maven) to record output fingerprints
+buildinfo=${artifactId}-${version}.buildinfo
+
+# if the release is finally not reproducible, link to an issue tracker entry if one was created or diffoscope result
+diffoscope=${artifactId}-${version}.diffoscope
```
Comment (hboutemy) [inline content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec:37]: this part has to be commented out, as there is no issue in your rebuild

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec~ [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# see instructions https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/doc/BUILDSPEC.md
```
Comment (hboutemy) [inline content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec~:1]: bad auto-save file committed

**Review by hboutemy — CHANGES_REQUESTED** [review]
just a few updates to do, please, for me to merge

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec (hunk 2 above)
Comment (jskov) [reply content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec]: I assume just the diffoscope line (I left issue= which is also present in reproducible-gradle)

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec~ (hunk 3 above)
Comment (jskov) [reply content/dk/mada/style/mada-style-gradle/mada-style-gradle-1.0.0.buildspec~]: duh!

File: content/dk/mada/style/mada-style-gradle/mada-style-gradle-0.9.9.buildinfo (hunk 1 above)
Comment (jskov) [reply content/dk/mada/style/mada-style-gradle/mada-style-gradle-0.9.9.buildinfo:28]: righto

**Comment by jskov** [comment]
> > Updates the comparison for 0.9.9: please re-run it. It works for me.
> 
> It seems you are lucky with 0.9.9: your build got a patch level of JDK that generates the same JavaScript in Javadoc. I was not as lucky as you. But IIUC, you were as unlucky as me with 0.9.8 :)
> 
> this is a global weakness for environment spec to rebuild javadoc that expands to `.module` files generated by Gradle: #193 created to ignore these `.module` files...
> 
> I'll merge your lucky build as part of this PR

Ah! So that is the reason.

I was looking into it during a weekend before my summer vacation, and over a couple of hours it suddenly was fixed.
Probably because my machine and the builder image got in sync.
Thanks!

**Comment by hboutemy** [comment]
thank you for your help

**Pull request merged**